node/test/tasklib.ts

```diff
@@ -564,6 +596,32 @@ describe('Test vsts-task-lib', function() {
 
             done();
         })
+
+        // filePathSupplied tests
```

Comment:
I just moved these two tests. I was trying to figure out what was covered already.

The tests were largely grouped already by the function being tested. Only these two were out of place.